src/listeners/discord/ready.ts

```diff
@@ -22,6 +23,7 @@ const event: DiscordEvent<"ready"> = {
       if (!client.user) return
       logger.info(`Bot [${client.user.username}] is ready`)
       ChannelLogger.ready(client)
+      UsageTrackingLogger.ready(client)
```

Comment:
any specific reason for this?
why don't put this inside `UsageTrackingLogger` constructor()

Reply:
put here as the same as ChannelLogger for the same pattern